app.py

```diff
@@ -9,9 +9,10 @@
 CLIENT_SECRET = '<your_client_secret>'
 
 api = MakeLeapsAPI(client_id=CLIENT_ID, client_secret=CLIENT_SECRET)
-token = api._auth_client()
+token = api.auth_client()
+print(token)
```

Comment:
It's a good idea to not get in the habit of committing things like print statements into your code.